tests/ui/methods.rs

```diff
@@ -22,6 +24,9 @@
     clippy::useless_format
 )]
 
+#[macro_use]
+extern crate option_helpers;
```

Comment:
Is this `extern crate` necessary? Aren't we using the 2018 edition in the tests? Or do we need this because of the `aux-build` of compiletest-rs?
```rust
use option_helpers::opt_map;
```
should be possible.

Reply:
Currently yes. I think it's an issue with compiletest: https://github.com/laumann/compiletest-rs/issues/150